NCS.DSS.AzureSearchUtility/Helpers/FileHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.IO;
+
+namespace NCS.DSS.AzureSearchUtility.Helpers
+{
+    public static class FileHelper
+    {
+        public static string GenerateSwaggerFileName(string environmentName)
+        {
+            return string.Format("{0}{1}{2}{3}{4:yyyyMMdd-hhmmss}{5}", "dss-", environmentName, "-searchutility", "-fa_swagger-def_", DateTime.Now, ".json");
+        }
+
+        public static string GenerateFilePath(string fileName)
+        {
+            if (string.IsNullOrEmpty(fileName))
+                throw new ArgumentNullException(nameof(fileName));
```

Comment:
Multiple: Could be useful to add some context to the argument null message to indicate which function was called. Maybe nameof(function_name) in the error message.